madmom/evaluation/key.py

```diff
@@ -134,14 +243,27 @@ class KeyEvaluation(EvaluationMixin):
         ('error_category', 'Error Category')
     ]
 
-    def __init__(self, detection, annotation, strict_fifth=False, name=None,
+    error_scores = {'correct': 1.0,
+                    'fifth': 0.5,
+                    'relative': 0.3,
+                    'relative_of_fifth': 0.0,
+                    'parallel': 0.2,
+                    'other': 0.0}
+
+    def __init__(self, detection,
+                 annotation,
+                 strict_fifth=False,
+                 name=None,
+                 relative_of_fifth=False,
                  **kwargs):
         self.name = name or ''
         self.detection = key_label_to_class(detection)
         self.annotation = key_label_to_class(annotation)
-        self.score, self.error_category = error_type(
-            self.detection, self.annotation, strict_fifth
-        )
+        self.error_category = error_type(self.detection,
+                                         self.annotation,
+                                         strict_fifth,
+                                         relative_of_fifth)
+        self.score = self.error_scores.get(self.error_category, 0.0)
```

Comment:
It would be better to use indexing here so that this line throws an error if an error category is unknown.

Reply:
OK, done